investment-app/TRINV.Api/TRINV.Domain/Entities/NewsComment.cs

```diff
@@ -0,0 +1,30 @@
+﻿namespace TRINV.Domain.Entities;
+
+using System.ComponentModel.DataAnnotations;
+
+public class NewsComment
+{
+    public NewsComment()
+    {
+        this.UpVote = 0;
```

Comment:
the same here remove constructor. Default value of interger is 0

Reply:
Done.